src/Http/PoolFactory.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Http;
+
+use GuzzleHttp\Pool;
+use Magento\MagentoCloud\App\ContainerInterface;
+use Magento\MagentoCloud\Util\UrlManager;
+use Psr\Http\Message\RequestInterface;
+
+/**
+ * Create Guzzle request pools.
+ */
+class PoolFactory
+{
+    /** @var ContainerInterface */
```

Comment:
Don't one-line phpdocs

Reply:
Why not? They're clean and easy to read.

Reply:
This violates coding standards, which ECE-Tools currently uses

Reply:
It's valid PHPDoc format and is already used in many places throughout both ECE Tools and other Magento packages (mostly in unit test classes). I can find nothing in https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html to suggest it violates our coding standards.